### ISSUE TITLE
Questions on coding

### ISSUE THREAD

**Issue by hddy2000** [AlexanderGuillermoSeguraBallesteros/RRL#2]
Dear Alexander,
I'm reading your codes, simultaneously with the paper, and get a question:
There is an 'eta' in the paper, that described as the magnitude of influence of the return Rt on the Sharp Ratio St,(in formula 15)but this parameter seems not to be included in your code.I assume you set the eta into 1, did you?

Eddie,

**Comment by hddy2000** [comment]
Here's the link to the moody's paper I mentioned, wanna make sure we are on the same page:
http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.1.7210&rep=rep1&type=pdf

**Comment by AlexanderGuillermoSeguraBallesteros** [comment]
That's correct, in the code uploaded I don't use the differential Sharpe Ratio. It is implemented as batch learning.

**Comment by hddy2000** [comment]
Hi,I actually find out that by using the 'rewards' you are assuming eta to be 0,not 1,which is exactly as formula 16 described:)
Have you considered working out a version with eta,and make it tunable? It could be very fun of that!

**Comment by luckykanwar** [comment]
Hi @hddy2000 and @AlexanderGuillermoSeguraBallesteros,

Its great to find a working code sample for this work. 

I am working on a Master's thesis in the area of Deep Reinforcement Learning and wasn't able to get my head around how to implement the RRL algorithm. 

I would greatly appreciate if you guys could give me some ideas on how to improve this model.

Thanks.
Nitin
